rasa/utils/train_utils.py

```diff
@@ -155,40 +161,87 @@ def _replace_deprecated_option(
     config: Dict[Text, Any],
     warn_until_version: Text = NEXT_MAJOR_VERSION_FOR_DEPRECATIONS,
 ) -> Dict[Text, Any]:
-    if old_option in config:
-        if isinstance(new_option, str):
-            rasa.shared.utils.io.raise_deprecation_warning(
-                f"Option '{old_option}' got renamed to '{new_option}'. "
-                f"Please update your configuration file.",
-                warn_until_version=warn_until_version,
-            )
-            config[new_option] = config[old_option]
-        else:
-            rasa.shared.utils.io.raise_deprecation_warning(
-                f"Option '{old_option}' got renamed to "
-                f"a dictionary '{new_option[0]}' with a key '{new_option[1]}'. "
-                f"Please update your configuration file.",
-                warn_until_version=warn_until_version,
-            )
-            option_dict = config.get(new_option[0], {})
-            option_dict[new_option[1]] = config[old_option]
-            config[new_option[0]] = option_dict
+    if old_option not in config:
+        return {}
+
+    if isinstance(new_option, str):
+        rasa.shared.utils.io.raise_deprecation_warning(
+            f"Option '{old_option}' got renamed to '{new_option}'. "
+            f"Please update your configuration file.",
+            warn_until_version=warn_until_version,
+        )
```

Comment:
why are their code changes in a documentation PR?

Reply:
because documentation needs config changes, config changes needs code changes. I cannot switch back and forth